os/net/lwip/src/api/sockets.c

```diff
@@ -393,7 +393,12 @@ struct lwip_sock *get_socket(int s, pid_t pid)
 	list = tcb->group->tg_socketlist;
 	sock = (struct lwip_sock *)list.sl_sockets[s].sock;
 
-	if (!sock || !sock->conn) {
+	if (!sock) {
+		LWIP_DEBUGF(SOCKETS_DEBUG, ("get_socket(%d): Socket is not in this thread's group.\n", s + LWIP_SOCKET_OFFSET));
+		set_errno(EBADF);
```

Comment:
I think why sock is null would have various reason. So just logging  "sock is null" is enough

Reply:
Modified. Thanks.